underpants.go

```diff
@@ -98,6 +98,10 @@ type conf struct {
 		// A special group, `*`, may be specified which allows any authenticated
 		// user.
 		AllowedGroups []string `json:"allowed-groups"`
+
+		// Fix the host header on the request to the backend proxied service to match
+		// what came in through the front door.
+		FixHost bool
```

Comment:
I think the property here should be a little more descriptive. How about `pass-through-host-header` or `retain-host-header`. Also be sure to add this directive to one of the sample json files.